src/cryptography/fernet.py

```diff
@@ -146,13 +150,16 @@ def __init__(self, fernets):
         self._fernets = fernets
 
     def encrypt(self, msg):
-        return self._fernets[0].encrypt(msg)
+        return self.encrypt_at_time(msg, int(time.time()))
+
+    def encrypt_at_time(self, msg, current_time):
+        return self._fernets[0].encrypt_at_time(msg, current_time)
 
     def rotate(self, msg):
         timestamp, data = Fernet._get_unverified_token_data(msg)
         for f in self._fernets:
             try:
-                p = f._decrypt_data(data, timestamp, None)
+                p = f._decrypt_data(data, timestamp, None, None)
```

Comment:
The [partial coverage](https://codecov.io/gh/pyca/cryptography/compare/eceea9d912994dbd84bfc7b27031c1254e675e28...3d1fadd7ffd2490df956aca725971db623e0ac15/src/tests/test_fernet.py?before=tests/test_fernet.py) looks like it's occurring because `time.time` is no longer called when `rotate` is called. This is because we now pass `None` for current_time, which is fine since we also pass a ttl of `None` so no comparison ever occurs.

The purpose of that test is to confirm that a `rotate` doesn't touch the timestamp when re-encrypting, so we can just modify it to check that via a call to `encrypt_at_time` and not monkeypatch at all.

Reply:
Hah, you're obviously right – I failed to notice that in the Codecov interface. This is addressed now.